source/common/network/io_socket_handle_impl.cc

```diff
@@ -305,7 +285,7 @@ Address::InstanceConstSharedPtr maybeGetDstAddressFromHeader(const cmsghdr& cmsg
     ipv6_addr->sin6_family = AF_INET6;
     ipv6_addr->sin6_addr = info->ipi6_addr;
     ipv6_addr->sin6_port = htons(self_port);
-    return getAddressFromSockAddrOrDie(ss, sizeof(sockaddr_in6), fd);
+    return Address::getAddressFromSockAddrOrDie(ss, sizeof(sockaddr_in6), fd);
```

Comment:
do you need the `Address::` qualification here and below? We didn't before.

Reply:
Yes, we move it to a different namespace.